packages/collector/test/tracing/cloud/azure/blob/app.js

```diff
@@ -0,0 +1,325 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+/* eslint-disable no-console */
+
+'use strict';
+
+require('../../../../..')();
+
+const { BlobServiceClient, BlobBatchClient, BlobClient, StorageSharedKeyCredential } = require('@azure/storage-blob');
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const port = require('../../../../test_util/app-port')();
+
+const app = express();
+const logPrefix = `Express / azure blob App (${process.pid}):\t`;
+const fs = require('fs');
+
+const filePath = `${__dirname}/sample.pdf`;
+const localFilePath = `${__dirname}/out.pdf`;
+const data1 = fs.readFileSync(filePath);
+const accountKey = process.env.ACC_KEY;
+const connStr = process.env.CONN_STR;
+const storageAccount = process.env.STORAGE_ACC;
```

Comment:
[The variable names need to be changed as well.](https://github.com/instana/nodejs/pull/967#discussion_r1429915939)

Reply:
resolved